docs/extensions/storage/_index.md

```diff
@@ -8,39 +8,144 @@ geekdocFilePath: _index.md
-| 9143      | gateway debug                                 |
-| 9144      | users                                         |
-| 9145      | users debug                                   |
-| 9146      | authbasic                                     |
-| 9147      | authbasic debug                               |
-| 9148      | authbearer                                    |
-| 9149      | authbearer debug                              |
-| 9150      | sharing                                       |
-| 9151      | sharing debug                                 |
-| 9152      | storage root                                  |
-| 9153      | storage root debug                            |
-| 9154      | storage home                                  |
-| 9155      | storage home debug                            |
-| 9156      | storage home data                             |
-| 9157      | storage home data debug                       |
-| 9158      | storage eos                                   |
-| 9159      | storage eos debug                             |
-| 9160      | storage eos data                              |
-| 9161      | storage eos data debug                        |
-| 9162      | storage oc                                    |
-| 9163      | storage oc debug                              |
-| 9164      | storage oc data                               |
-| 9165      | storage oc data debug                         |
-| 9166-9177 | reserved for s3, wnd, custom + data providers |
-| 9178      | storage public link                           |
-| 9179      | storage public link data                      |
+## Architecture Overview
+
+The below diagram shows the ocis services and the contained reva services within as dashed boxes. In general:
+1. A request comes in at the proxy and is authenticated using oidc.
+2. It is forwarded to the ocis frontend which handles ocs and ocdav requests by talking to the reva gateway using the CS3 API.
+3. The gateway acts as a facade to the actual CS3 services: storage providers, user providers, group providers and sharing providers.
+
+{{< svg src="extensions/storage/static/overview.drawio.svg" >}}
+
+The dashed lines in the diagram indicate requests that are made to authenticate requests or lookup the storage provider:
+1. After authenticating a request the proxy may either use the CS3 `userprovider` or the accounts service to fetch the user information that will be minted into the `x-access-token`.
+2. The gateway will verify the JWT signature of the `x-access-token` or try to authenticate the request itself, eg. using a public link token.
+
+{{< hint warning >}}
+The bottom part is lighter because we will deprecate it in favor of using only the CS3 user and group providers after moving some account functionality into reva and glauth. The metadata storage is not registered in the reva gateway to prevent seperate user metadata necessary for running the service from data that is being served directly.
```

Comment:
`to prevent seperate` sounds like an error to me, unclear what this sentence suggests :)

Reply:
Also `eg.` again, perhaps best to search&replace with `e.g.` in the whole file